english/interact/whip/whip.md

```diff
@@ -0,0 +1,27 @@
+You whip the rope over your head as hard as you can.
+
+It crashes into the wall with a thud.
+
+Nothing happens.
+
+You hit the wall with your rope again.
+
+Silence.
+
+Then...
+
+BLOP! BLOP! BLOP!
+
+BLOP! BLOP! BLOP!
+
+There are now SIX marshmallow hands in front of you!
+
+And it looks like you made them all angry!
+
+Do you:
+
+[Lay down and play dead] (/english/macgyver/toss-lighter/Dead/Play_dead.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.